site/src/docs/patterns/cookies/basics.mdx

```diff
@@ -0,0 +1,114 @@
+| **Content presentation**      | Cookies are used to display various types of content to the user.                                                              | Can be considered necessary if the cookie is strictly related to presenting the content and not gathering data.                                                                                                                                                                              | Monitoring how the user is using specific content                                            |
+| **Location**                  | Location cookies can be used to utilise location information (GPS) received from the user’s device.                            | Never necessary.                                                                                                                                                                                                                                                                             | Accurate GPS location                                                                        |
+| **Load balancing**            | Load-balancing cookies ensure that the service loads and works quickly and efficiently.                                        | Always necessary (as they ensure that the service is operating smoothly).                                                                                                                                                                                                                    | A client identifying hash to enable better routing                                                                                            |
+[Table 3:Cookie categories]|
+
+#### Can I use a custom category?
+
+If your cookies do not fit into any of the pre-defined categories, you are allowed to use custom categories.
+
+#### We only have a handful of cookies. Do we still have to categorize them?
+
+Yes, cookies always have to be put into a category. If you have a small number of cookies you can consider grouping them into a single category. Also, if most of the cookies are related to a single third-party service, you can group them per service (E.g. "Service: YouTube").
+
+### Prepare your CookieConsent component
+
+Now that you have gathered and categorised a list of cookies for your service, it is a time to learn how the HDS CookieConsent is used. To read more about the component, please refer to the <Link href="/patterns/cookies/using-the-cookieconsent">Using the CookieConsent page</Link>.
```

Comment:
The link does not work on demo environment. Use this syntax instead:

`[Using the CookieConsent page](/patterns/cookies/using-the-cookieconsent)`

Reply:
Fixed